index.js

```diff
@@ -14,14 +14,15 @@ module.exports = function (options) {
 
   return function send ({endpoint, tokens, config, cb}) {
     const _config = configFactory(config)
+    const _tokens = tokens || []
```

Comment:
Would it not be better to leverage ES6 default parameters?

Reply:
ok no probs

Reply:
updated

Reply:
Looks like the builder needs updating to put the `,` in the correct place, sorry about that 😞